docs/resources/metal_device_network_type.md

```diff
@@ -2,7 +2,10 @@
 subcategory: "Metal"
 ---
 
-# equinix_metal_device_network_type (Resource)
+# equinix_metal_device_network_type (Resource, Deprecated)
```

Comment:
There doesn't seem to be a standard for marking entire resources as deprecated in the docs.  The [HashiCorp recommendations](https://developer.hashicorp.com/terraform/plugin/sdkv2/best-practices/deprecations#provider-data-source-or-resource-removal) are focused only on the code.

Reply:
Actually the differentiation between resource / datasource in parentheses seems to be a thing of the past, I don't see it in the documentation of any other provider. For me it's fine like this, but the documentation generation plugin won't put things like that either if we stop using templates in the future.

Reply:
So the **Note** below might be enough, but again, I think the way you put it seems clarifying